pkg/storage/fs/ocis/node.go

```diff
@@ -469,3 +481,18 @@ func (n *Node) ReadGrant(ctx context.Context, grantee string) (g *provider.Grant
 	}
 	return e.Grant(), nil
 }
+
+func (n *Node) hasUserShares(ctx context.Context) bool {
+	g, err := n.ListGrantees(ctx)
+	if err != nil {
+		appctx.GetLogger(ctx).Error().Err(err).Msg("hasUserShares: listGrantees")
+		return false
+	}
+
+	for i := range g {
+		if strings.Contains(g[i], grantPrefix+"u") {
```

Comment:
`"u"` seems like a magic string. It might be because I'm disconnected from this part of the code but it tells me nothing about why `g[i]` has to contain a `prefix + u`.

I think readability could benefit of abstracting the semantics of `"u"` to a package level var with a more descriptive name. WDYT?

Reply:
Yeah, you are right. Will change it. :+1:

Reply:
thank you! it'd also give me some more context :)